<?xml version="1.0" ?>
<!--This does not replace URDF, and is not an extension of URDF.
    This is a format for representing semantic information about the robot structure.
    A URDF file must exist for this robot as well, where the joints and the links that are referenced are defined
-->
<robot name="kuka_lbr_iiwa_14_r820">
    <!-- Manipulators -->
    <group name="manipulator">
        <chain base_link="base_link" tip_link="tool0" />
    </group>

    <!-- Groups kinematic plugins -->
    <kinematics_plugin_config filename="package://tesseract_ros_examples/config/kuka_lbr_iiwa_14_r820_plugins.yaml"/>

    <!--DISABLE COLLISIONS By default it is assumed that any link of the robot could potentially come into collision with any other link in the robot. This tag disables collision checking between a specified pair of links. -->
    <disable_collisions link1="base_link" link2="link_1" reason="Adjacent" />
    <disable_collisions link1="base_link" link2="link_2" reason="Never" />
    <disable_collisions link1="base_link" link2="link_3" reason="Never" />
    <disable_collisions link1="base_link" link2="link_4" reason="Never" />
    <disable_collisions link1="link_1" link2="link_2" reason="Adjacent" />
    <disable_collisions link1="link_1" link2="link_3" reason="Never" />
    <disable_collisions link1="link_1" link2="link_4" reason="Never" />
    <disable_collisions link1="link_1" link2="link_5" reason="Never" />
    <disable_collisions link1="link_1" link2="link_6" reason="Never" />
    <disable_collisions link1="link_1" link2="link_7" reason="Never" />
    <disable_collisions link1="link_2" link2="link_3" reason="Adjacent" />
    <disable_collisions link1="link_2" link2="link_4" reason="Never" />
    <disable_collisions link1="link_2" link2="link_5" reason="Never" />
    <disable_collisions link1="link_2" link2="link_6" reason="Never" />
    <disable_collisions link1="link_2" link2="link_7" reason="Never" />
    <disable_collisions link1="link_3" link2="link_4" reason="Adjacent" />
    <disable_collisions link1="link_3" link2="link_5" reason="Never" />
    <disable_collisions link1="link_3" link2="link_6" reason="Never" />
    <disable_collisions link1="link_3" link2="link_7" reason="Never" />
    <disable_collisions link1="link_4" link2="link_5" reason="Adjacent" />
    <disable_collisions link1="link_4" link2="link_6" reason="Never" />
    <disable_collisions link1="link_4" link2="link_7" reason="Never" />
    <disable_collisions link1="link_5" link2="link_6" reason="Adjacent" />
    <disable_collisions link1="link_5" link2="link_7" reason="Never" />
    <disable_collisions link1="link_6" link2="link_7" reason="Adjacent" />
</robot>
